sensu/gohandler_test.go

```diff
@@ -55,12 +55,14 @@ var (
 func TestNewGoHandler(t *testing.T) {
 	values := &handlerValues{}
 	options := getHandlerOptions(values)
-	goHandler := NewGoHandler(&defaultHandlerConfig, options, func(event *types.Event) error {
+	goPlugin := NewGoHandler(&defaultHandlerConfig, options, func(event *types.Event) error {
 		return nil
 	}, func(event *types.Event) error {
 		return nil
 	})
 
+	goHandler := goPlugin.(*goHandler)
```

Comment:
Assert there is no error on the type assertion:
```suggestion
	goHandler, err := goPlugin.(*goHandler)
	assert.Nil(t, err)
```

However, the type assertion should not be necessary if the struct satisfies the interface.

Reply:
The type assertion actually returns a bool, not an error but this was a good suggestion. I updated the PR to include it :-)

Reply:
Whoops yeah you're right! Thanks for updating that 👍